### ISSUE TITLE
Alternate log file behavior is confusing or possibly a bug

### ISSUE THREAD

**Issue by SamWhited** [profanity-im/profanity#1467]
When specifying an alternate log file location with `-f` or `--logfile` I naively expect to be able to set a path with any file I choose at the end, however, instead it only works if I don't use a path and it appends `.log` to the end no matter what I do and sticks it in the `logs` tree.

## Expected Behavior

If I run `profanity -f /tmp/myfile` a log file will be created (or opened if it exists) at `/tmp/myfile`.

## Current Behavior

If I run `profanity -f /tmp/myfile` a log file is not created at all (silently).
If I run `profanity -f myfile` a logfile is created at `$XDG_DATA_HOME/profanity/logs/myfile.log` which is not the file I specified (though this behavior is slightly more understandable than if I specify an absolute full file path).

## Possible Solution

Please consider respecting the absolute path of the log file I specified and not appending ".log" as I may be trying to open an existing log file.

## Context

I am trying to have profanity write logs to a named pipe that is open in another program which will ship any logs written to the pipe elsewhere.

**Comment by jubalh** [comment]
@SamWhited which version of Profanity are you running?

**Comment by SamWhited** [comment]
Oops, sorry, left out the last section, that would probably be helpful:

```
$ profanity -v
Profanity, version 0.9.5
Copyright (C) 2012 - 2019 James Booth <boothj5web@gmail.com>.
Copyright (C) 2019 - 2020 Michael Vetter <jubalh@iodoru.org>.
License GPLv3+: GNU GPL version 3 or later <https://www.gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Build information:
XMPP library: libmesode
Desktop notification support: Enabled
OTR support: Enabled (libotr 4.1.1)
PGP support: Enabled (libgpgme 1.15.0)
OMEMO support: Enabled
C plugins: Enabled
Python plugins: Enabled (3.9.1)
GTK icons: Disabled
```

**Comment by jubalh** [comment]
Then I think this has been fixed in 0.10.0 (released two days ago).
See https://github.com/profanity-im/profanity/issues/1442.

**Comment by SamWhited** [comment]
Perfect timing! 0.10.0 is already in the Arch repos so I updated and it appears to be working, thank you.
